hip/preconditioner/isai_kernels.hip.cpp

```diff
@@ -168,6 +168,24 @@ GKO_INSTANTIATE_FOR_EACH_VALUE_AND_INDEX_TYPE(
     GKO_DECLARE_ISAI_GENERATE_EXCESS_SYSTEM_KERNEL);
 
 
+template <typename ValueType, typename IndexType>
+void scale_excess_solution(std::shared_ptr<const DefaultExecutor>,
```

Comment:
Do you anywhere document your strategy, like what you do with the long-row systems?